taxi/views.py

```diff
@@ -33,6 +34,20 @@
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
 
+class ManufacturerCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Manufacturer
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:manufacturer-list")
+    template_name = "taxi/manufacturer_form.html"
+
+
+class ManufacturerDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Manufacturer
+    template_name = "taxi/manufacturer_confirm_delete.html"
+    success_url = reverse_lazy("taxi:manufacturer-list")
```

Comment:
The `ManufacturerDeleteView` is correctly implemented with all necessary attributes.